### PR TITLE
StaticLLMPipeline: Update config

### PR DESCRIPTION
**Pull request by TolyaTalamanov**

### CODE REVIEW

File: src/cpp/src/llm_pipeline_static.cpp [hunk 1]
```diff
@@ -254,13 +247,15 @@ ov::AnyMap get_baseline_common_config() {
 
 ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
     auto config = get_baseline_common_config();
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_SLICE_OUT", "YES");
+    }
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:252]: SLICE_OUT can be enabled unconditionally, why only CW?

File: src/cpp/src/llm_pipeline_static.cpp [hunk 2]
```diff
@@ -271,8 +266,8 @@ ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
         return { };
     }
     auto config = get_default_common_config(model);
-    if (get_option<std::string>(config, "NPUW_FUNCALL_FOR_ALL").value_or("NO") == "YES") {
-        config.emplace("NPUW_PARALLEL_COMPILE", "YES");
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_DQ", "YES");
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:270]: If we don't enable DQ (see `else`), I'd also recommend to disable PMM (`PMM : NO`)

File: src/cpp/src/llm_pipeline_static.cpp (hunk 1 above)
Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp:252]: Thought there were regressions for this case, will change

Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp]: Done

File: src/cpp/src/llm_pipeline_static.cpp (hunk 2 above)
Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp:268]: Done

File: src/cpp/src/llm_pipeline_static.cpp [hunk 3]
```diff
@@ -260,37 +255,33 @@ ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
...
-        config.emplace("NPU_DPU_GROUPS", npudesc->max_tiles);
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_DQ", "YES");
+    } else {
+        config.emplace("NPUW_PMM", "NO");
+        if (npudesc.has_value() &&
+            npudesc->arch == "4000" &&
+            npudesc->max_tiles != -1) {
+            config.emplace("NPU_DPU_GROUPS", npudesc->max_tiles);
+        }
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:272]: you set DPU groups here only if it is GQ.

In fact, this setting should be done regardless of the Q type.

**Review by dmatveev — CHANGES_REQUESTED** [review]

File: src/cpp/src/llm_pipeline_static.cpp (hunk 3 above)
Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp]: Done

File: src/cpp/src/llm_pipeline_static.cpp [hunk 4]
```diff
@@ -260,19 +255,16 @@ ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
...
-    enable_npuw_dq_if_allowed(config, model);
     return config;
 }
 
 ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
                                       const std::optional<NPUDesc>& npudesc) {
-    // NB: Empty config for devices != NPU
-    if (!npudesc.has_value()) {
-        return { };
-    }
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:272]: is it ok?

**Review by dmatveev — APPROVED** [review]
Let's do this!

File: src/cpp/src/llm_pipeline_static.cpp (hunk 4 above)
Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp:272]: Decided not to fallback into CPU but user NPUW:CPU instead

**Pull request merged**